code/modules/antagonists/hivemind/spells/hivemind_dominance.dm

```diff
@@ -0,0 +1,46 @@
+/obj/effect/proc_holder/spell/self/hive_dominance
+	name = "One Mind"
+	desc = "Our true power... finally within reach."
+	panel = "Hivemind Abilities"
+	charge_type = "charges"
+	charge_max = 1
+	invocation_type = "none"
+	clothes_req = 0
+	human_req = 1
+	action_icon = 'icons/mob/actions/actions_hive.dmi'
+	action_background_icon_state = "bg_hive"
+	action_icon_state = "assim"
+	antimagic_allowed = TRUE
+
+/obj/effect/proc_holder/spell/self/hive_dominance/cast(mob/living/user = usr)
+	var/datum/antagonist/hivemind/hive = user.mind.has_antag_datum(/datum/antagonist/hivemind)
+	if(!hive)
+		to_chat(user, "<span class='notice'>This is a bug. Error:HIVE1</span>")
+		return
+	hive.glow = mutable_appearance('icons/effects/hivemind.dmi', "awoken", -BODY_BEHIND_LAYER)
+	for(var/datum/antagonist/hivevessel/vessel in hive.avessels)
+		var/mob/living/carbon/C = vessel.owner?.current
+		C.Jitter(15)
+		C.Unconscious(150)
+		to_chat(C, "<span class='boldwarning'>Something's wrong...</span>")
+		addtimer(CALLBACK(GLOBAL_PROC, /proc/to_chat, C, "<span class='boldwarning'>...your memories are becoming fuzzy.</span>"), 45)
+		addtimer(CALLBACK(GLOBAL_PROC, /proc/to_chat, C, "<span class='boldwarning'>You try to remember who you are...</span>"), 90)
+		addtimer(CALLBACK(GLOBAL_PROC, /proc/to_chat, C, "<span class='assimilator'>There is no you...</span>"), 110)
+		addtimer(CALLBACK(GLOBAL_PROC, /proc/to_chat, C, "<span class='bigassimilator'>...there is only us.</span>"), 130)
+		addtimer(CALLBACK(C, /atom/proc/add_overlay, hive.glow), 150)
+
+	for(var/datum/antagonist/hivemind/enemy in GLOB.hivehosts)
+		if(enemy.owner)
+			enemy.owner.RemoveSpell(new/obj/effect/proc_holder/spell/self/hive_dominance)
+			var/mob/living/carbon/C = enemy.owner?.current
+			if(!enemy.hiveID == hive.hiveID)
```

Comment:
This logic is flawed, !enemy.hiveID will be 0 if hiveID is anything but 0 and 1 otherwise. You are then comparing 0/1 to the hiveID